docker-compose.yml

```diff
@@ -28,3 +26,15 @@ services:
       - DB_PORT=${DB_PORT}
```

Comment:
![image](https://github.com/user-attachments/assets/b3bd08bd-491b-4551-8335-24a0a85db05e)

The endpoints are not mapped in nestJS application, I cannot perform simple registering

![image](https://github.com/user-attachments/assets/58df93d4-19c8-4540-a089-07e8d6b9ae37)

I would suggest testing simple endpoints to see whether docker files are working or not. I see no relation between backend & mongo inside docker. The only thing is that the db is initialized first (depend_on property in dockero-compose file), before backend.

Reply:
Of course changes have been tested and it works like a charm for me 😉.

Creating a user:
![image](https://github.com/user-attachments/assets/381a3be6-1c43-43fa-a900-2d20666eafb6)

Get on users:
![image](https://github.com/user-attachments/assets/31569228-b2b6-40ec-a55f-a57e9a038ba1)

I think it proves that everything works and you must be doing something wrong.


Following the [Installation Guide](https://github.com/OBELISK-TEAM/OBELISK/blob/OK-65/docs/INSTALL.md) should fix the problems. I suppose you don't have correct envs set.

Reply:
my mistake, old images were used